### PR TITLE
Resources menu

### PR DESCRIPTION
**Pull request by BowgartField**
* Fix header css
* Hide header on responsive
* Add "resources" and "language" menu's item

### CODE REVIEW

File: components/HeaderDesktop.tsx [hunk 1]
```diff
@@ -11,45 +11,40 @@ import FrenchFlag from './svg/flags/FrenchFlag'
...
+      }}
     >
       <Link href="/">
         <a>
           <img src="/logo/polarfox.png" alt="Polarfox logo" />
         </a>
       </Link>
       <div
-        className="flex items-center space-x-20 font-semibold"
+        className="grid grid-flow-col items-center justify-end text-center"
```
Comment (Klemah) [inline components/HeaderDesktop.tsx:30]: Readd `font-semibold`

**Comment by Klemah** [comment]
Small details that need to be fixed:

- only display one dropdown at a time
- add the flag next to "languages": see Figma
- in "languages", we should not display "Languages" but rather the selected language (in this case, English)

File: components/HeaderDesktop.tsx [hunk 2]
```diff
@@ -148,18 +203,25 @@ interface ProductProps {
...
+  const call = () => {
+    console.log('ok')
+  }
+
   return (
     <div style={{ width: calcRem(137) }}>
       <Link href={href}>
         <a
-          className="flex rounded-xl hover:bg-gray-dark mx-3 p-3 pt-4"
+          className={classNames(buttonHover ? 'bg-gray-dark' : '','flex w-full rounded-xl hover:bg-gray-dark p-3 pt-4')}
```
Comment (Klemah) [inline components/HeaderDesktop.tsx:217]: Replace `buttonHover ? 'bg-gray-dark' : ''` with `{ 'bg-gray-dark': buttonHover }`

File: components/HeaderDesktop.tsx [hunk 3]
```diff
@@ -58,85 +53,145 @@ export function HeaderDesktop({ className }: SectionProps) {
...
+    setTimeout(function () {
+      setMenuOpened(false)
+    }, 200)
+  }
+
   return (
-    <div
+    <div>
+      <div
+        className={classNames(menuOpened || locked ? 'bg-gray-mid2' : '', 'grid items-center hover:bg-gray-mid2 rounded-3xl px-6 py-4 gap-x-2')}
```
Comment (Klemah) [inline components/HeaderDesktop.tsx:81]: Replace `menuOpened || locked ? 'bg-gray-mid2' : ''` with `{ 'bg-gray-mid2': menuOpened || locked }`

File: components/HeaderDesktop.tsx [hunk 4]
```diff
@@ -58,85 +53,145 @@ export function HeaderDesktop({ className }: SectionProps) {
...
+        style={{ gridTemplateColumns: '0.9fr 0.1fr' }}
+        onMouseEnter={() => setMenuOpened(true)}
+        onMouseLeave={() => hideMenu()}
+      >
+        {name}
+        <MenuArrow />
+      </div>
+      <div
+        className={classNames(menuOpened || locked ? '' : 'hidden', className)}
```
Comment (Klemah) [inline components/HeaderDesktop.tsx:90]: Same as above

File: components/HeaderMobile.tsx [hunk 5]
```diff
@@ -81,120 +77,119 @@ export function HeaderMobile({
...
-          href="https://akita.network"
-          linkTitle="Visit site"
-        >
-          AKITA Network
-        </Product>
+        {name}
+        <MenuArrow style={{ width: calcRem(12), height: calcRem(8) }} />
       </div>
+      
+      <div className={classNames(menuOpened ? '': 'hidden')}>
```
Comment (Klemah) [inline components/HeaderMobile.tsx:166]: Replace `menuOpened ? '': 'hidden'` with `{ 'hidden': menuOpened }`

File: components/HeaderDesktop.tsx [hunk 6]
```diff
@@ -11,45 +11,40 @@ import FrenchFlag from './svg/flags/FrenchFlag'
...
           fontSize: calcRem(14),
           lineHeight: calcRem(16.45)
         }}
       >
         <ProductsDropdown />
-        {/* TODO: Remove the div below */}
-        <div />
         <Link href="/about">
-          <a>About</a>
+          <a className="px-6">About</a>
```
Comment (Klemah) [inline components/HeaderDesktop.tsx:38]: Idea: refactor code to remove `px-6` and use `gap-x-6` or `space-x-6` instead (or `12`)

File: components/HeaderDesktop.tsx [hunk 7]
```diff
@@ -170,56 +232,46 @@ function Product({ logo, href, linkTitle, children }: ProductProps) {
...
-        }
-      `}</style>
-    </div>
+    <Dropdown
+      name="Language"
+      className="absolute block grid mt-1 grid-flow-row p-4 gap-y-2 rounded-3xl z-1"
+      rightAlignment="0.5%"
+    >
+      <Language href="/">
+        {' '}
```
Comment (Klemah) [inline components/HeaderDesktop.tsx:263]: Can this `{' '}` be replaced with some padding?

File: components/HeaderDesktop.tsx [hunk 8]
```diff
@@ -232,12 +284,13 @@ function Language({ href, children }: LanguageProps) {
   return (
     <Link href={href}>
       <a
-        className="flex items-center justify-between rounded-xl hover:bg-gray-dark p-3 pt-4 font-semibold"
+        className="grid auto-rows-fr items-center text-left rounded-xl hover:bg-gray-dark p-3 pt-4 font-semibold w-full"
```
Comment (Klemah) [inline components/HeaderDesktop.tsx:287]: `text-left` is the default behavior. Perhaps we should get rid of `text-center` at line 30 instead of using `text-left` here.

Comment (Klemah) [reply components/HeaderDesktop.tsx:287]: Same with other occurrences of `text-left`.

File: components/HeaderMobile.tsx [hunk 9]
```diff
@@ -50,10 +50,8 @@ export function HeaderMobile({
             className="flex flex-col justify-between z-5 font-semibold mt-4 overflow-hidden"
             style={{ fontSize: calcRem(24), lineHeight: calcRem(58) }}
           >
-            <div className="flex justify-between items-center">
-              Products
-              <MenuArrow style={{ width: calcRem(12), height: calcRem(8) }} />
-            </div>
+
```
Comment (Klemah) [inline components/HeaderMobile.tsx:53]: Delete empty line

File: components/HeaderMobile.tsx [hunk 10]
```diff
@@ -63,10 +61,8 @@ export function HeaderMobile({
             <Link href="/faq">
               <a>FAQ</a>
             </Link>
-            <div className="flex justify-between items-center">
-              Resources
-              <MenuArrow style={{ width: calcRem(12), height: calcRem(8) }} />
-            </div>
+            <ResourcesAccordionMenu/>
+
```
Comment (Klemah) [inline components/HeaderMobile.tsx:65]: Delete empty line

File: components/HeaderMobile.tsx [hunk 11]
```diff
@@ -81,120 +77,119 @@ export function HeaderMobile({
...
+    <AccordionMenu name="Resources">
+
+      <AccordionMenuItem
+        href="/tutorials"
+        linkTitle="Visit site"
+        name="Tutorials"
+      />
+
+      <AccordionMenuItem
+        href="/Polarfox Litepaper V2.6.pdf"
```
Comment (Klemah) [inline components/HeaderMobile.tsx:131]: This link appears four times in the app (and it's not the only one suffering from this).

How about this: we can move all the links to a constant file, so it's easier for us to update.

(This should be done in another pull request).

File: components/HeaderMobile.tsx [hunk 12]
```diff
@@ -81,120 +77,119 @@ export function HeaderMobile({
...
+        linkTitle="Read"
+        name="Litepaper"
+      />
+
+      <AccordionMenuItem
+        href="https://github.com/Polarfox-DEX"
+        linkTitle="Examine"
```
Comment (Klemah) [inline components/HeaderMobile.tsx:138]: Replace with "View"

**Review by Klemah — CHANGES_REQUESTED** [review]

File: components/HeaderMobile.tsx (hunk 5 above)
Comment (Klemah) [reply components/HeaderMobile.tsx:166]: Also remove the empty line

File: components/HeaderMobile.tsx (hunk 11 above)
Comment (Klemah) [reply components/HeaderMobile.tsx:131]: Also remove empty line

File: components/HeaderMobile.tsx (hunk 12 above)
Comment (Klemah) [reply components/HeaderMobile.tsx:138]: Also remove empty line

File: components/HeaderDesktop.tsx (hunk 6 above)
Comment (BowgartField) [reply components/HeaderDesktop.tsx:38]: I don't see how i can change this because padding need to be applied only on <Link>

File: components/HeaderDesktop.tsx (hunk 8 above)
Comment (BowgartField) [reply components/HeaderDesktop.tsx:287]: If i delete `text-left` the text is no more at left

File: components/HeaderDesktop.tsx (hunk 6 above)
Comment (Klemah) [reply components/HeaderDesktop.tsx:38]: Just tried it on my machine.

You can add `gap-x-14`, remove `px-6` here, and remove `px-6` at line 75 (in the `Dropdown()` function). This produces a result that is very similar.

File: components/HeaderDesktop.tsx (hunk 8 above)
Comment (Klemah) [reply components/HeaderDesktop.tsx:276]: You need to remove `text-center` at line 30

Comment (Klemah) [reply components/HeaderDesktop.tsx:276]: Then, you can remove all instances of `text-left`, I believe.

**Review by Klemah — APPROVED** [review]
